actions/telegramaction.py

```diff
@@ -10,7 +10,8 @@ class TelegramAction(BasicAction):
     """Action to send a Telegram message to a certain user or group"""
     name = "TelegramAction"
 
-    def __init__(self, token, receiver, custom_payload=None, template=None):
+    def __init__(self, token, receiver, custom_payload=None, template=None,
+                 proxies=None):
```

Comment:
I think the singleton would be a good idea since the `proxies` dict is very personnalizable (it is possible to use proxy for only some hosts, or to use the `no_proxy` key).